test/export/test_export.py

```diff
@@ -2939,6 +2939,32 @@ def forward(self, x):
                 },
             )
 
+    def test_unbacked_slice_forward(self):
```

Comment:
it will be nice to check the graph content in the test here. or add that to the PR summay at least